web/content/index.adoc

```diff
@@ -6,6 +6,7 @@ title: Home
 
 The following releases are supported, meaning they receive (security) updates. Visit the https://community.theforeman.org/c/support/10[support forum] for questions.
 
+* link:/release/3.7/[Foreman 3.8 & Katello 4.10]
 * link:/release/3.7/[Foreman 3.7 & Katello 4.9]
 * link:/release/3.6/[Foreman 3.6 & Katello 4.8]
```

Comment:
Only when it's GA. At this point it's only a release candidate. It should be noted that we don't use the `web` directory in the 3.8 branch so there's no point in changing it here. You should change it in `master`. In 163a9efbcba2887f5ac96708a5a481f8dadf561e you can see how I did it last time: explicitly list it as an RC instead of a supported release.

Reply:
Ok I already did that in a previous PR then https://github.com/theforeman/foreman-documentation/commit/ddba2d718d91ade9bdfd3375f875cc883993449a